modules/draco/src/lib/draco-parser.js

```diff
@@ -318,9 +318,11 @@ export default class DracoParser {
    */
   _deduceAttributeName(attributeData, options) {
     const {extraAttributes = {}} = options;
```

Comment:
This only triggers if options is not present or undefined. The `null`/`falsy` case can be handled with

```
const extraAttributes = options.extraAttributes || {};
```

I think this is the only change that is needed.

Reply:
Yes that can work! The check I added of `typeof extraAttributes === 'object'` would also prevent a crash in case the attributes are not passed as an object, but perhaps that's not needed, or at least it should throw a readable exception.